recipes_source/recipes/timer_quick_start.py

```diff
@@ -0,0 +1,461 @@
+#       26975  ???:_int_malloc [/usr/lib64/libc-2.28.so]
+#       20400  ???:__tls_get_addr [/usr/lib64/ld-2.28.so]
+#       19800  /data/users/taylorrobie/repos/pytorch/build/../aten/src/ATen/TensorIterator.cpp:at::TensorIte ... ers/taylorrobie/miniconda3/envs/ab_ref/lib/python3.7/site-packages/torch/lib/libtorch_cpu.so]
+#       13500  ???:malloc [/usr/lib64/libc-2.28.so]
+#       11300  /data/users/taylorrobie/repos/pytorch/build/../c10/util/SmallVector.h:at::TensorIteratorBase::fast_set_up(at::TensorIteratorConfig const&)
+#       10000  /data/users/taylorrobie/repos/pytorch/build/../aten/src/ATen/TensorIterator.cpp:at::TensorIte ... ers/taylorrobie/miniconda3/envs/ab_ref/lib/python3.7/site-packages/torch/lib/libtorch_cpu.so]
+#        9200  ???:free [/usr/lib64/libc-2.28.so]
+#        9200  /data/users/taylorrobie/repos/pytorch/build/../aten/src/ATen/record_function.cpp:at::shouldRu ... ers/taylorrobie/miniconda3/envs/ab_ref/lib/python3.7/site-packages/torch/lib/libtorch_cpu.so]
+#         ...
+#         100  /data/users/taylorrobie/repos/pytorch/build/../c10/util/FunctionRef.h:at::native::(anonymous namespace)::mul_kernel(at::TensorIterator&)
+#         100  /data/users/taylorrobie/repos/pytorch/build/../c10/util/FunctionRef.h:at::TensorIteratorBase::for_each(c10::function_ref<void (char**, long const*, long)>, long)
+#         100  /data/users/taylorrobie/repos/pytorch/build/../c10/util/ArrayRef.h:at::TensorIteratorBase::get_data_ptrs(c10::ArrayRef<char*>, c10::ArrayRef<long>) const
+#         100  /data/users/taylorrobie/repos/pytorch/build/../c10/core/impl/LocalDispatchKeySet.h:c10::impl::ExcludeDispatchKeyGuard::ExcludeDispatchKeyGuard(c10::DispatchKeySet)
+#         100  /data/users/taylorrobie/repos/pytorch/build/../c10/core/Storage.h:c10::TensorImpl::release_resources()
+#         100  /data/users/taylorrobie/repos/pytorch/build/../c10/core/ScalarType.h:c10::computeDispatchKey(c10::optional<c10::ScalarType>, c10::optional<c10::Layout>, c10::optional<c10::Device>)
+#         100  /data/users/taylorrobie/repos/pytorch/build/../aten/src/ATen/core/dispatch/DispatchKeyExtract ... (at::Tensor const&, at::Tensor const&)> const&, at::Tensor const&, at::Tensor const&) const'2
+#         100  /data/users/taylorrobie/repos/pytorch/build/../aten/src/ATen/core/dispatch/DispatchKeyExtract ... r (at::Tensor const&, at::Tensor const&)> const&, at::Tensor const&, at::Tensor const&) const
+#         100  /data/users/taylorrobie/repos/pytorch/build/../aten/src/ATen/core/boxing/impl/WrapFunctionInt ... const&, at::Tensor const&)>::call(c10::OperatorKernel*, at::Tensor const&, at::Tensor const&)
+#
+#     Total: 587972
+#
+
+###############################################################################
+# There's some useful information here. The most expensive sections of the code
+# are malloc, free, and various PyTorch utilities. However there is also a lot
+# of extraneous path junk cluttering things up. For that, you can call the
```

Comment:
junk? Also, probably explain what `as_standardized` does right away, and don't show output without it.

Reply:
Done. (And yes, not a great way of phrasing.)